docs/aks-installation.md

```diff
@@ -101,3 +109,61 @@ you should be able to see these Pods running in your cluster:
     pod "metrics-server-85c57978c6-pwzcx" deleted
     pod "tunnelfront-649ff5fb55-5lxg7" deleted
     ```
+
+## Deploy Antrea to an AKS Engine cluster
+
+Antrea is an integrated CNI of AKS Engine, and can be installed in
+`networkPolicyOnly` mode or `encap` mode to an AKS Engine cluster as part of the
+AKS Engine cluster deployment. To learn basics of AKS Engine cluster deployment,
+please refer to [AKS Engine Quickstart Guide](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/quickstart.md).
```

Comment:
please refer to the

Reply:
"AKS Engine Quickstart Guide" is the title of the document.